datalake/src/test/java/com/sequenceiq/datalake/service/validation/cloudstorage/CloudStorageValidatorTest.java

```diff
@@ -97,4 +100,40 @@ public void validateEnvironmentRequestCloudStorageValidation() {
         ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> underTest.validate(new CloudStorageRequest(), environment, validationResultBuilder));
         assertFalse(validationResultBuilder.build().hasError());
     }
+
+    @Test
+    public void validateBackupLocation() {
+        when(environment.getCredential()).thenReturn(new CredentialResponse());
+        when(secretService.getByResponse(any())).thenReturn("secret");
+        when(regionAwareInternalCrnGenerator.getInternalCrnForServiceAsString()).thenReturn("crn");
+        when(regionAwareInternalCrnGeneratorFactory.iam()).thenReturn(regionAwareInternalCrnGenerator);
+        when(credentialToCloudCredentialConverter.convert(any())).thenReturn(
+                new CloudCredential("id", "name", Map.of("secretKey", "thisshouldnotappearinlog"), "acc", false));
+
+        when(cloudProviderServicesV4Endopint.validateObjectStorage(any())).thenReturn(new ObjectStorageValidateResponse());
+        ValidationResultBuilder validationResultBuilder = new ValidationResultBuilder();
+        ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> underTest.validateBackupLocation(new CloudStorageRequest(), environment, validationResultBuilder));
+        assertFalse(validationResultBuilder.build().hasError());
+    }
```

Comment:
Please verify the contents of the request passed to `cloudProviderServicesV4Endopint.validateObjectStorage()` using an `ArgumentCaptor`.